en/contributing/release.md

```diff
@@ -5,28 +5,40 @@ These are the instructions on how to get a release out the door.
 > **Note** The idea is of course to automate this as much as possible!
 
 1. Check if all open pull requests are merged, that need to go in.
-2. Check that CI on `develop` passed.
-3. [Generate the docs](build.md#build_api_reference) and update the reference docs.
-4. Check or update the examples in the docs.
-5. Merge `develop` into `master` and tag the release:
+1. Check that CI on `develop` passed.
+1. [Generate the docs](build.md#build_api_reference) and update the reference docs.
+1. Check or update the examples in the docs.
+1. Run link checker and fix any breaks.
+1. Merge `develop` into `master`:
```

Comment:
Is it normal that it's all `1.`? I guess it is, since it got changed everywhere :thinking:

Reply:
That's a feature of markdown. If you don't want to bother with actually numbering things you can just use `1.`.